AppCenter/AppCenter/Internals/Storage/MSLogDBStorage.m

```diff
@@ -40,27 +40,29 @@ - (instancetype)init {
 
 #pragma mark - Save logs
 
-- (BOOL)saveLog:(id<MSLog>)log withGroupId:(NSString *)groupId {
+- (BOOL)saveLog:(id<MSLog>)log withGroupId:(NSString *)groupId critical:(BOOL)critical {
```

Comment:
Just thinking about it now but we know the real time feature will come and will we have the need  to select RealTime logs directly from the storage? Just a comment not a blocker it can evolve in the future.